pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -17,20 +17,20 @@
 
 buildPythonPackage rec {
   pname = "tensorflow";
-  version = "0.10.0";
+  version = "1.0.0";
   name = "${pname}-${version}";
   format = "wheel";
 
   src = fetchurl {
     url = if stdenv.isDarwin then
-      "https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-${version}-py2-none-any.whl" else
-      "https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${version}-cp27-none-linux_x86_64.whl";
+      "https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-${version}-py3-none-any.whl" else
+      "https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${version}-cp35-cp35m-linux_x86_64.whl";
     sha256 = if stdenv.isDarwin then
```

Comment:
Could you please leave the 2.7 version in? Like this: https://github.com/NixOS/nixpkgs/pull/20794/commits/69c9298259c21ef52c41b3e6f9f1b760ffa8ee41